src/js/modules/comments.js

```diff
@@ -0,0 +1,41 @@
+const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const appID = '9JcQwe8YeiV9ciPMfpK9';
+
+const commentsHeader = document.querySelector('.comments-container h4');
+const spinner = document.querySelector('#comments-spinner');
+
+const getComments = async (id) => {
+  commentsHeader.innerHTML = 'Comments (0)';
+  spinner.innerHTML = '<i class="fas fa-spinner fa-spin"></i>';
+  await fetch(`${baseURL}${appID}/comments?item_id=${id}`)
```

Comment:
###Suggestion
This can just use the network class  `new NetWorkRequest(baseURL).get('/comments?item_id=${id}', true).then((json) => { //print to screen })`